module.modulemap

```diff
@@ -0,0 +1,12 @@
+// This file is required by current CocoaPods (0.39.0). 
+//  - Generates warning: Umbrella header for module 'SQLite' does not include header 'fts3_tokenizer.h'
```

Comment:
I wonder if we can make this a private header in the module to avoid the warning?